lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -102,6 +104,40 @@ static const std::string kUSDCylinderPrimImage { "out_USD_Cylinder.png" };
 static constexpr char    kUSDSpherePrimItem[] = "Sphere";
 static constexpr char    kUSDSpherePrimLabel[] = "Sphere";
 static const std::string kUSDSpherePrimImage { "out_USD_Sphere.png" };
+static constexpr char    kAllRegisteredTypesItem[] = "All Registered";
+static constexpr char    kAllRegisteredTypesLabel[] = "All Registered";
+
+// Grouping and name mapping for registered schema plugins
+static const std::vector<std::string> kSchemaPluginNames = {
+    "usdGeom",
+    "usdLux",
+    "mayaUsd_Schemas",
+    "usdMedia",
+    "usdRender",
+    "usdRi",
+    "usdShade",
+    "usdSkel",
+    "usdUI",
+    "usdVol",
+    "AL_USDMayaSchemasTest",
+    "AL_USDMayaSchemas",
+};
+// clang-format off
+static const std::vector<std::string> kSchemaNiceNames = {
+    "Geometry",
+    "Lighting",
+    "Maya Reference",
+    "Media",
+    "Render",
+    "RenderMan",
+    "Shading",
+    "Skeleton",
+    "UI",
+    "Volumes",
+    "", // Skip legacy AL schemas
+    "", // Skip legacy AL schemas
+};
+// clang-format on
```

Comment:
All of this will be unused if usd version is 2008 or less. So I think it should be wrapped in the if PXR_VERSION as well. You might get an unused warning (which will be an error since we have strict mode on) when building with older USD version.